en/docs/guides/identity-lifecycles/update-profile.md

```diff
@@ -101,7 +101,7 @@ admin services, see [Calling Admin Services](insert-calling admin services).
 ----
 
 !!! info "Related Topics"
-    - [Concept: User Profiles](TODO:insert-link-to-concept)
```

Comment:
let's add something about what a user profile is also somewhere in the concept doc?

Reply:
Details regarding the user account are added to the user management doc. Not sure if we need to add user profile information separately. Let's discuss this separately?